tests/models/test_gptq_marlin.py

```diff
@@ -0,0 +1,96 @@
+"""Compares the outputs of gptq vs gptq_marlin 
+Note: GPTQ and Marlin do not have bitwise correctness.
+As a result, in this test, we just confirm that the top selected tokens of the
+Marlin/GPTQ models are in the top 3 selections of each other.
+Note: Marlin internally uses locks to synchronize the threads. This can
+result in very slight nondeterminism for Marlin. As a result, we re-run the test
+up to 3 times to see if we pass.
+Note: This test currently fails running with --forked with the following:
+    RuntimeError: Cannot re-initialize CUDA in forked subprocess.
+    To use CUDA with multiprocessing, you must use the 'spawn' start method
+Run `pytest tests/models/test_gptq_marlin.py`.
+"""
+import os
+
+import pytest
+import torch
+from compare_utils import check_logprobs_close
```

Comment:
It would be better to do `from .compare_utils import check_logprobs_close` here (or rather `from .utils import check_logprobs_close`) after moving -- for consistency with the rest of the code and I'm also not sure what this does if there is a global compare_utils module, or if this is called from a different directory that might have a compare_utils.py file itself.